core/src/banking_stage.rs

```diff
@@ -1306,26 +1493,29 @@ impl BankingStage {
             qos_service,
         );
         process_tx_time.stop();
-        let unprocessed_tx_count = unprocessed_tx_indexes.len();
-        inc_new_counter_info!(
-            "banking_stage-unprocessed_transactions",
-            unprocessed_tx_count
-        );
```

Comment:
Is this captured elsewhere now? I guess rebuffered packets count?

Reply:
yeah should be captured in the rebuffered count